MsCorePkg/Include/Library/MacAddressEmulationPlatformLib.h

```diff
@@ -0,0 +1,60 @@
+/** @file
+
+  Defines the APIs for the MAC Address Emulation Platform library.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _MAC_ADDRESS_EMULATION_PLATFORM_LIB_H_
+#define _MAC_ADDRESS_EMULATION_PLATFORM_LIB_H_
+
+/**
+  Reports whether MacEmulation is enabled, and returns the address to emulate.
+
+  @param[out]  Address - Pointer to a buffer to receive the emulated MAC address to use for this platform.
+
+  @retval  EFI_UNSUPPORTED - Feature is not enabled/supported by the platform.
+  @retval  EFI_STATUS      - Status of other calls in the function.
+
+**/
+EFI_STATUS
+IsMacEmulationEnabled (

```

Comment:
It seems like this function should return a `BOOLEAN` instead of a status. Or, be renamed something like `GetMacEmulationAddress()`.